### PR TITLE
updated views and widgets to not require asset packages

### PR DESCRIPTION
**Pull request by schmunk42**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | yes
| Tests pass?   | ?

Related https://github.com/yiisoft/yii-console/issues/12#issuecomment-443163191

I've just uncommented the Bootstrap widgets as an example.

The Alert is currently outputting JSON content, this is an example for content that could be wrapped in a JS variable and function as initial "Server Side Rendered" data. This should be easily usable with native JS, jQuery, Vue or any other JS-lib.

Contact and Login are currently not working, but I think that's a separate issue atm.

### CODE REVIEW

**Comment by schmunk42** [comment]
Looks like this atm:
bildschirmfoto von 2018-11-30 16-05-19
bildschirmfoto von 2018-11-30 16-05-28

For sure we can supply some basic CSS also in a later step.

File: src/controllers/SiteController.php [hunk 1]
```diff
@@ -60,6 +61,7 @@ public function actions()
      */
     public function actionIndex()
     {
+        $this->app->getSession()->addFlash('info', 'Hello World!');
```
Comment (samdark) [inline src/controllers/SiteController.php:64]: Maybe it's better to display something like "Hello, it is flash info message."?

File: src/views/layouts/main.php [hunk 2]
```diff
@@ -27,7 +27,7 @@
 <?php $this->beginBody() ?>
 
 <div class="wrap">
-    <?php
+    <?php /* example for bootstrap 4 navbar, if yiisoft/yii-bootstrap4 is installed
```
Comment (samdark) [inline src/views/layouts/main.php:30]: Not sure we need that...

File: src/views/site/index.php [hunk 3]
```diff
@@ -18,34 +20,35 @@
...
             <div class="col-lg-4">
-                <h2><?= $this->app->t('yii-base-web', 'Heading') ?></h2>
+                <h2><?= $this->app->t('yii-base-web', 'Info') ?></h2>
 
-                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
-                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
-                    ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
-                    fugiat nulla pariatur.</p>
+                <p>
+                    This is the Index page. You may modify the following file to customize its content:
```
Comment (samdark) [inline src/views/site/index.php:26]: Good idea to have it instead of gibberish...

File: src/views/site/index.php [hunk 4]
```diff
@@ -18,34 +20,35 @@
...
+                </p>
 
-                    <p><a class="btn btn-default" href="http://www.yiiframework.com/doc/"><?= $this->app->t('yii-base-web', 'Yii Documentation') ?> &raquo;</a></p>
-            </div>
-            <div class="col-lg-4">
-                <h2><?= $this->app->t('yii-base-web', 'Heading') ?></h2>
+                <code><?= __FILE__ ?></code>
```
Comment (samdark) [inline src/views/site/index.php:29]: Should be HTML-encoded.

File: src/widgets/Alert.php [hunk 5]
```diff
@@ -57,16 +60,16 @@ public function run()
                 continue;
             }
 
-            foreach ((array) $flash as $i => $message) {
-                echo \yii\bootstrap4\Alert::widget([
-                    'body' => $message,
-                    'closeButton' => $this->closeButton,
-                    'options' => array_merge($this->options, [
+            echo "<data id='{$this->id}'>";
```
Comment (samdark) [inline src/widgets/Alert.php:63]: Widget should not echo. It should return.

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by schmunk42** [comment]
Sorry I accidentally merged this, while pushing to `master` (used wrong remote).
Comments should be fixed, let me know if there are other issues.

**Comment by schmunk42** [comment]
See https://github.com/yiisoft/yii-base-web/commit/8013f2f60903835ae59a13b8985b694f616e7227

**Pull request merged**